.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Not sure if this is likely (yet) but we've got shell scripts that call `git commit`

Do we want to skip the hooks when `CI=true` is set in the GitHub workflows?

```suggestion
. "$(dirname -- "$0")/_/husky.sh"

# Ignore hook in GitHub workflows
[ -n "$CI" ] && exit 0
```

Reply:
That'd likely save some execution time on those commits on the CI. I'd prefer doing that in a blanket way, either using the [`HUSKY=0` env variable](https://typicode.github.io/husky/#/?id=with-env-variables) to not have to manage each script individually. 

If we ever find some checks that need to run before CI Git operations, I think it'd be better that they're explicitely shown in what the CI does rather than happening through git hooks anyway.

Reply:
Yeah don't mind how, just know we have `git config` and `git diff` calls here:
https://github.com/alphagov/govuk-frontend/blob/git-hook-linting/.github/workflows/diff-change-to-dist.yaml

They don't follow an npm install so just being safe. Cheers @romaricpascal

Reply:
I don't think those would trigger any hooks, [judging by the list in Git docs](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks). I'm still keep to deactivate Husky's hooks altogether on CI, though. Cheers for pointing the use of Git out 😄